v2/src/translations/en_US.json

```diff
@@ -43,11 +63,18 @@
         "topTokenTitle": "Top Tokens",
         "favoriteTokenTitle": "Favorite Tokens",
         "marketTokenTitle": "Token Market",
+        "totalAddress": "Total Addresses",
+        "tokenBalance": "Token Balance",
+        "tokenTotal": "total tokens",
+        "portfolioEmpty": "Your portfolio is empty",
+        "portfolioValue": "Portfolio Value",
         "header": {
             "blockNumber": "Block #",
+            "watchList": "Watchlist",
             "time": "Timestamp",
             "transactions": "Transactions",
             "miner": "Miner",
+            "balance": "Balances",
```

Comment:
you have a string 'Balance' already in common. no need to define another one with 'Balances', you can plural